src/mlpack/tests/ann_layer_test-mod.cpp

```diff
@@ -0,0 +1,2937 @@
+/**
+ * @file mlpack/tests/ann_layer_test-mod.cpp
```

Comment:
What is this change , I am not sure about this change

Reply:
Ahh, nice catch, that's something I didn't mean to check in!  Fixed in 700e5ee72.